Rotation matrix output columns aligned.

malibrud opened the pull request:
Hello,

I love this library.  I changed the format of the output rotation matrix so that the columns align.  To do this I added a function to convert to fixed width strings of each matrix element.

Mark Brudnak

malibrud committed:
Changed the output for the rotation matrix so that the columns are aligned.

gaschler commented on index.html:
```diff
@@ -171,7 +171,10 @@
     var r = m.elements;
-    var s = '[ '+toReal(r[0])+', '+toReal(r[4])+', '+toReal(r[8])+';\n'+toReal(r[1])+', '+toReal(r[5])+', '+toReal(r[9])+';\n'+toReal(r[2])+', '+toReal(r[6])+', '+toReal(r[10])+' ]';
+    var s = 
+         '[ ' +pmOneToStr(r[0],7)+', '+pmOneToStr(r[4],7)+', '+pmOneToStr(r[8],7)+';\n'
```
Since you only use this function with argument "7", you can drop the argument and make 7 default.

malibrud replied:
I hard coded the 7.

gaschler commented on index.html:
```diff
@@ -331,6 +334,18 @@
 		return x;
 	}
 }
+function pmOneToStr(x, p){
```
pmOneToStr is cryptic, use toOutputString or prettyNumber instead.

malibrud replied:
I changed name to 'toFixedWidth'

gaschler commented on index.html:
```diff
@@ -331,6 +334,18 @@
+	if (!isNaN(parseFloat(x)) && isFinite(parseFloat(x)))
+	{
+		var s = x.toFixed(p);
+                if (x >=0) s = ' ' + s;
```
space missing ">= 0"

malibrud replied:
I added the space.

gaschler reviewed (changes requested):
Thank you very much for the contribution.
I have only minor remarks before it can be merged.

malibrud committed:
Made requested changes to pull request #2.
1) Changed pmOneToStr to toFixedWidth
2) Hard coded the precision number rather than passing it as a parameter.
3) Added a space on line 341 between the '>=' and the '0'

malibrud committed:
Merge branch 'feature/rotationMatrixFormat'

# Conflicts:
#	index.html

malibrud commented:
I made all requested changes.

gaschler commented:
Thanks again for your patch, merged it.
